docs/contributing.md

```diff
@@ -0,0 +1,165 @@
+# Contributing to JAX
+
+Everyone can contribute to JAX, and we value everyone's contributions. There are several
+ways to contribute, including:
+
+- Answering questions on JAX's [discussions page](https://github.com/google/jax/discussions)
+- Improving or expanding JAX's [documentation](http://jax.readthedocs.io/)
+- Contributing to JAX's [code-base](http://github.com/google/jax/)
+- Contributing in any of the above ways to the broader ecosystem of [libraries built on JAX](https://github.com/google/jax#neural-network-libraries)
+
+The JAX project follows [Google's Open Source Community Guidelines](https://opensource.google/conduct/).
+
+## Ways to contribute
+
+We welcome pull requests, in particular for those issues marked with
+[contributions welcome](https://github.com/google/jax/issues?q=is%3Aopen+is%3Aissue+label%3A%22contributions+welcome%22) or
+[good first issue](https://github.com/google/jax/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).
+
+For other proposals, we ask that you first open a Github
+[Issue](https://github.com/google/jax/issues/new/choose) or
+[Discussion](https://github.com/google/jax/discussions)
+to seek feedback on your planned contribution.
+
+## Contributing code using Pull Requests
+
+We do all of our development using git, so basic knowledge is assumed.
+
+Follow these steps to contribute code:
+
+1. Fork the JAX repository by clicking the 'Fork' button on the
+   [repository page](http://www.github.com/google/jax). This create a copy
+   of the Flax repository in your own account.
```

Comment:
Suggestion:

- **Button** (emphasis added for UI elements, I think Google Cloud use that)
- Flax -> JAX (🤷‍♂️ )
- "creates"

```suggestion
1. Fork the JAX repository by clicking the **Fork** button on the
   [repository page](http://www.github.com/google/jax). This creates a copy
   of the JAX repository in your own account.
```

Reply:
Done - thanks!

Reply:
No, thank you @jakevdp  👍